kola/harness.go

```diff
@@ -371,6 +371,7 @@ func runTest(h *harness.H, t *register.Test, pltfrm string) {
 		NoSSHKeyInUserData: t.HasFlag(register.NoSSHKeyInUserData),
 		NoSSHKeyInMetadata: t.HasFlag(register.NoSSHKeyInMetadata),
 		NoEnableSelinux:    t.HasFlag(register.NoEnableSelinux),
+		DebugSystemdUnits:  strings.Split(Options.DebugSystemdUnits, " "),
```

Comment:
If the intent is to make this a user-provided global setting (as the commit message indicates), this doesn't belong in `RuntimeConfig`, which is a per-cluster config driven by test declarations.  Note that the option is currently kola-wide but only actually applies to subcommands calling through `runTest`, i.e. not `spawn`.  A better approach would be to have `NewBaseCluster` take `Options` rather than `Options.BaseName`, as it should have in the first place, and then let `RenderUserData` read it directly from there.